pkgs/by-name/xa/xarcan/package.nix

```diff
@@ -85,6 +85,10 @@ stdenv.mkDerivation (finalPackages: {
     xtrans
   ];
 
+  patches = [
+    ./0001-fix-segfault.patch
```

Comment:
Add a `# comment` why the patch is needed.

https://github.com/jtojnar/nixpkgs-hammering/blob/main/explanations/missing-patch-comment.md

Reply:
Done